downloader/downloader.go

```
@@ -694,8 +694,18 @@ func (downloader *Downloader) Download(data *extractors.Data) error {
 	if !downloader.option.Silent {
 		fmt.Printf("Merging video parts into %s\n", mergedFilePath)
 	}
+
 	if stream.Ext != "mp4" || stream.NeedMux {
-		return utils.MergeFilesWithSameExtension(parts, mergedFilePath)
+		err = utils.MergeFilesWithSameExtension(parts, mergedFilePath)
+	} else {
+		err = utils.MergeToMP4(parts, mergedFilePath, title)
+	}
+	if err != nil {
+		return nil
+	}
```

Why do we ignore this error directly?

---

...this is a mistake.